scripts/react-16-install-prep.mjs

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
these are all versions which can't be easily changed through the yarn cli like we do with react, react-dom at the root level, but it's just helping to tie down certain versions so we don't have duplicates installed and they are all versions which work with the react version we've installed